include/nnstreamer-edge.h

```diff
@@ -65,14 +65,21 @@ typedef enum {
 } nns_edge_event_e;
 
 typedef enum {
-  NNS_EDGE_PROTOCOL_TCP = 0,
-  NNS_EDGE_PROTOCOL_UDP,
-  NNS_EDGE_PROTOCOL_MQTT,
-  NNS_EDGE_PROTOCOL_AITT,
-  NNS_EDGE_PROTOCOL_AITT_TCP,
+  NNS_EDGE_CONNECT_TYPE_TCP = 0,
+  NNS_EDGE_CONNECT_TYPE_UDP,
+  NNS_EDGE_CONNECT_TYPE_MQTT,
+  NNS_EDGE_CONNECT_TYPE_HYBRID,
 
-  NNS_EDGE_PROTOCOL_MAX
-} nns_edge_protocol_e;
+  NNS_EDGE_CONNECT_TYPE_UNKNOWN
+} nns_edge_connect_type_e;
+
+typedef enum {
+  NNS_EDGE_FLAG_NONE = 0,
+  NNS_EDGE_FLAG_RECV = (1 << 0),
+  NNS_EDGE_FLAG_SEND = (1 << 1),
+
```

Comment:
How about adding a flag to distinguish the server from the client?
```suggestion
NNS_EDGE_FLAG_SERVER = (1 << 2),
```

Reply:
And then remove `is_server` from `nns_edge_start`.

Reply:
agreed

Reply:
I added new flag for server and updated related source. Please review the changes.